### PR TITLE
chore: replace black / isort / flake by ruff

### PR DESCRIPTION
**Pull request by czosel**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -37,43 +37,30 @@ requests = "^2.31.0"
...
-  "snapshots",
-]
-known_first_party = "ebau_gwr"
-multi_line_output = 3
-include_trailing_comma = true
-force_grid_wrap = 0
-combine_as_imports = true
-line_length = 88
+[tool.ruff.lint]
+extend-select = ["I"]
```
Comment (czosel) [inline pyproject.toml:59]: this enforces isort-like behavior. We don't use this in inosca/ebau yet, but I think it makes sense?

Comment (czosel) [inline pyproject.toml:59]: this enforces isort-like behavior. We don't use this in inosca/ebau yet, but I think it makes sense?

Comment (Yelinz) [reply pyproject.toml:59]: makes sense

**Review by Yelinz — APPROVED** [review]
just needs a rebase

**Comment by czosel** [comment]
@Yelinz done

**Pull request merged**